dlp/snippets/deid.py

```diff
@@ -937,6 +937,79 @@ def deidentify_with_replace_infotype(project, item, info_types):
 # [END dlp_deidentify_replace_infotype]
 
 
+# [START dlp_deidentify_simple_word_list]
+def deidentify_with_simple_word_list(
+    project,
+    input_str,
+    info_type,
+    word_list
+):
```

Comment:
Missing return annotation 
```suggestion
) -> None:
```

Reply:
okay the same is not followed in other code samples.

Reply:
ok, updated.